server/pa.js

```diff
@@ -2,12 +2,43 @@
 const Validator = require('express-validator')
 
 // In-house dependencies
-const { helpers, twilioHelpers } = require('brave-alert-lib')
+const { helpers, googleHelpers, twilioHelpers } = require('brave-alert-lib')
 const db = require('./db/db')
 
 const paApiKeys = [helpers.getEnvVar('PA_API_KEY_PRIMARY'), helpers.getEnvVar('PA_API_KEY_SECONDARY')]
 const paPasswords = [helpers.getEnvVar('PA_PASSWORD_PRIMARY'), helpers.getEnvVar('PA_PASSWORD_SECONDARY')]
 
+const validateGetGoogleTokens = Validator.body(['authCode']).trim().notEmpty()
+
+async function getGoogleTokens(req, res) {
+  try {
+    const validationErrors = Validator.validationResult(req).formatWith(helpers.formatExpressValidationErrors)
+    if (validationErrors.isEmpty()) {
+      // paGetTokens will throw an error if the authorization code doesn't originate from PA
+      const tokens = await googleHelpers.paGetTokens(req.body.authCode)
+      res.status(200).json(tokens) // send tokens to PA: { accessToken, idToken }
+    }
+  } catch (error) {
+    helpers.log('PA: Unauthorized request to get Google tokens')
+    res.status(401)
+  }
+}
+
+const validateGetGooglePayload = Validator.body(['idToken']).trim().notEmpty()
+
+async function getGooglePayload(req, res) {
+  try {
+    const validationErrors = Validator.validationResult(req).formatWith(helpers.formatExpressValidationErrors)
+    if (validationErrors.isEmpty()) {
+      // paGetPayload will throw an error if the ID token is not valid (see brave-alert-lib for more details)
+      res.status(200).json(await googleHelpers.paGetPayload(req.body.idToken))
```

Comment:
This is just a comment on readability, so up to you if you want to bother updating anything. 
But it may make your code easier to read if you store `await googleHelpers.paGetPayload(req.body.idToken)` in a variable first, then pass the variable when calling `res.json()`. I think you follow this format on line 18-19 in `getGoogleTokens`. The downside is it adds an extra line, so again your call. :)

This is what I mean:
``` 
const payload = await googleHelpers.paGetPayload(req.body.idToken)
res.status(22).json(payload)
````

Reply:
good point!! I'll make this change.